src/javascript/binary/base.js

```diff
@@ -222,3 +222,13 @@ if (!/backoffice/.test(document.URL)) { // exclude BO
         LocalStore.set('active_loginid', match);
     });
 }
+
+
+//this method is to prevent caching on sensitive data
+pjax_config_page("new_account/real", function(){
```

Comment:
this looks like a convoluted way to add a tag.. Why can't you put the tag straight into the html?